### ISSUE TITLE
Should always check ninja in run() of BuildExtension?

### ISSUE THREAD

**Issue by Roy-Kid** [diegoferigo/cmake-build-extension#44]
https://github.com/diegoferigo/cmake-build-extension/blob/fea4503c3a77ca1169dd84f1c09b39b3b8760555/src/cmake_build_extension/build_extension.py#L92

In this line, I wonder if the ninja always be checked if I already specify another generator like Unix Makefiles?

PS: no matter what generator I specify, it always throw an error:
```
==> Configuring:
$ cmake -S /tmp/build-via-sdist-p3l2l10i/xtensor_python_test-0.0.1 -B /tmp/build-via-sdist-p3l2l10i/xtensor_python_test-0.0.1/build/temp.linux-x86_64-cpython-310_Pybind11Bindings -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=/tmp/build-via-sdist-p3l2l10i/xtensor_python_test-0.0.1/build/lib.linux-x86_64-cpython-310/mymath_pybind11 -DPython3_ROOT_DIR=/tmp/build-env-yxeuhf6l -DCALL_FROM_SETUP_PY:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=OFF -DEXAMPLE_WITH_SWIG:BOOL=OFF -DEXAMPLE_WITH_PYBIND11:BOOL=ON

==> Building:
$ cmake --build /tmp/build-via-sdist-p3l2l10i/xtensor_python_test-0.0.1/build/temp.linux-x86_64-cpython-310_Pybind11Bindings --config Release

==> Installing:
$ cmake --install /tmp/build-via-sdist-p3l2l10i/xtensor_python_test-0.0.1/build/temp.linux-x86_64-cpython-310_Pybind11Bindings

CMake Error: Could not create named generator "Unix Makefiles"
```

Thanks in advance if you can help me!

**Comment by diegoferigo** [comment]
Can you please check if after #45 your problem is now fixed?

**Comment by diegoferigo** [comment]
No response, closing. Should be fixed by #46.
